client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -27,6 +27,12 @@ import Gravatar from 'components/gravatar';
 import i18n from 'lib/mixins/i18n';
 
 /**
+ * Constants
+ */
+
+const STATS_PAGE = '/stats/insights/';
```

Comment:
I didn't know about that component... but for what I'm seeing, there's no way to append a new site slug to the link it provides. Does it worth to add that functionality to the component or is it making it too complex for the sake of some fringe cases like this one?